CHANGELOG.asciidoc

```diff
@@ -65,6 +65,7 @@ https://github.com/elastic/beats/compare/v6.0.0-beta2...master[Check the HEAD di
 - Change `MySQL active connections` visualization title to `MySQL total connections`. {issue}4812[4812]
 - Fix map overwrite in docker diskio module. {issue}5582[5582]
 - Fix connection leak in mongodb module. {issue}5688[5688]
+- Fix include top processes in case more processes included then existed. {pull}5729[5729]
```

Comment:
I think it's the other way around.

"Fix the include top N processes feature for cases where there are fewer processes than N."

Reply:
Isn't both the same just formulating it the other way around? Changing it to your option as it's easier to understand.

Reply:
Oh, I see what you were saying. The "more processes included" was misinterpreted by me. My bad.